translate/api/Tests/TestBatchTranslate.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System;
+using System.IO;
+using System.Collections.Generic;
+using System.Text;
+using Xunit;
+using Google.Cloud.Translate.V3.Samples;
+using GoogleCloudSamples;
+using Google.Cloud.Storage.V1;
+using Google.Apis.Storage.v1.Data;
+using System.Linq;
+
+namespace Tests
+{
+    public class BatchTranslateTest : IDisposable
+    {
+        protected string ProjectId { get; private set; } = Environment.GetEnvironmentVariable("GOOGLE_PROJECT_ID");
+        protected string _bucketName { get; private set; }
+        protected Bucket Bucket { get; private set; }
+        protected string InputUri { get; private set; } = "gs://cloud-samples-data/translation/glossary_ja.csv";
+        readonly CommandLineRunner _sample = new CommandLineRunner()
+        {
+            VoidMain = TranslateV3BatchTranslateTextMain.Main
+        };
+
+        // Setup
+        public BatchTranslateTest()
+        {
+            // Create temp bucket
+            var storageClient = StorageClient.Create();
+            _bucketName = "translate-v3-" + TestUtil.RandomName();
+            Bucket = storageClient.CreateBucket(ProjectId, _bucketName);
```

Comment:
You don't use Bucket again, you can delete the property and skip this assignment.